docs/sources/developers/plugins/sign-a-plugin.md

```diff
@@ -32,7 +32,16 @@ To verify ownership of your plugin, you need to generate an API key that you'll
 
 Public plugins need to be reviewed by the Grafana team before you can sign them.
 
-1. Submit your plugin for [review]({{< relref "publish-a-plugin/#publishing-your-plugin-for-the-first-time" >}})
+1. Submit your plugin for review. 
+
+   1. Sign in to [grafana.com](https://grafana.com)
+   1. On the My Account page, go to Org Settings.
+   1. On the Org Settings page, click **My plugins**. 
+   1. Click **Submit New Plugin**. 
+   1. Complete the form, and then click **Submit**.
```

Comment:
```suggestion
   1. Complete the form and click **Submit**.
```

Reply:
-1 for this recommended change, because it distinguishes two actions.

Reply:
On the fence about this. It is clear enough, but if the phrases were more complex, the original distinguishes between two separate actions.

Reply:
I use ", and then" out of habit for translatability, but I can use "and" if that's the standard here. Is there anything in our style guide to help? Perhaps it's just preferring concision to translatability.

Reply:
Yes, the main reason is that  in Chris's version it's not clear to a translator whether "click" is a. noun or a verb, and therrefore the sentence becomes ambiguous.